module/config/config.go

```diff
@@ -17,6 +17,8 @@ const (
 	bech32PrefixConsAddr = "gravityvalcons"
 	// bech32PrefixConsPub defines the bech32 prefix of a consensus node public key
 	bech32PrefixConsPub = "gravityvalconspub"
+
+	GravityEvmChainId = "417834"
```

Comment:
this needs it's own comment about preventing replay attacks.

Reply:
Updated